packages/@react-spectrum/s2/test/Combobox.test.tsx

```diff
@@ -163,4 +163,50 @@ describe('Combobox', () => {
       expect(option).toHaveAttribute('aria-posinset', `${index + 1}`);
     }
   });
+
+  it('should support contextual help', async () => {
+    // Issue with how we don't render the contextual help button in the fake DOM since PressResponder isn't using createHideableComponent
+    let warn = jest.spyOn(global.console, 'warn').mockImplementation();
```

Comment:
Gross that this is a thing, I remember discussing this last month or so but don't think we landed on anything...

Reply:
to be clear, it warns that the PressResponder doesn't have a button in the fake DOM